community/jmx/src/main/java/org/neo4j/jmx/impl/KernelBean.java

```diff
@@ -54,6 +54,25 @@
         this.instanceId = kernel.instanceId();
         this.query = support.createMBeanQuery( instanceId );
 
+        // perhaps not the most elegant way to tell which edition we are running...
+        String edition;
```

Comment:
we already have editionName in https://github.com/neo4j/neo4j/blob/3.0/community/kernel/src/main/java/org/neo4j/kernel/impl/factory/GraphDatabaseFacadeFactory.java#L95, and according to usages it's properly populated